client/views/add/add.js

```diff
@@ -92,13 +92,17 @@ Template.add.events({
     }
     Meteor.call('addMods', mods, template.data._id, (error, result) => {
       // If the result is an object, there was an error
-      if (typeof result === 'object') {
+      if (typeof result === 'object' && result.length > 0) {
         // Alert the error
         for (let i = 0; i < result.length; i++) {
           if (result[i].name === 'moderators') {
             showModsError('You can only assign 4 moderators per instance.');
             return false;
           }
+          if(result[i].name === 'owner') { // Check is the server returned error corresponding to the addition of owner as moderator
+            showModsError('You cannot add owner of the instance as moderator'); // Display the error message
```

Comment:
`You cannot add owner of the instance as moderator` > `You cannot add the owner of the instance as a moderator`